t/03-xsstatic.t

```diff
@@ -11,9 +11,6 @@ use MakeMaker::Test::Utils;
 use MakeMaker::Test::Setup::XS;
 use Test::More;
 
-plan skip_all => "Disabled as broken perl installs give false negative"
-  # if not static perl, and not author
-  unless !$Config{usedl} or $ENV{AUTHOR_TESTING};
```

Comment:
The rationale is that this patch fixes said false negatives. I've tested this with Alien::unibilium and Alien::SVN.